### ISSUE TITLE
Request for creating useHeading hook

### ISSUE THREAD

**Issue by emilov2501** [emilov2501/use-ui#5]
I'd like to be able to use a useHeading hook that would include:

- tests files
- hook file (entry point)
- accessibility
- basic styles (responsive)

**Comment by sachyshyn** [comment]
@emilov2501 is it the common component for h1-h6 tags? If yes, my proposal is to implement it like this API:

const Heading = useHeading({
    props: {
        level: number from 1 to 6, // default value might be 3
        className: string,
        content: jsx (eventually string) 
    },
    additionalProps: {
        // there should be other possible props like accessibility features or test ids
    }
});

**Comment by emilov2501** [comment]
Yeah, sure but also need to include user specific class with classnames, turn on or off ellipsis mode @sachyshyn
